ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -593,6 +593,26 @@ custom_platform_settings:
       - AWE-7220R
 ```
 
+### PREVIEW WAN HA Direct Link now uses Port-Channel by default
+
+For WAN direct HA, AVD will now configure a port-channel by default.
+This feature was introduced recently in EOS and may not be supported on your version.
+
+To retain the previous configuration use custom_platform_settings use `use_port_channel_for_direct_ha: false` under `wan_ha`.
+
+```diff
+  wan_router:
+    node_groups:
+      - group: Site42
+        cv_pathfinder_region: AVD_Land_West
+        cv_pathfinder_site: Site42
+        wan_ha:
+          enabled: true
++         use_port_channel_for_direct_ha: true
```

Comment:
This will not retain the old behavior :)

```suggestion
+         use_port_channel_for_direct_ha: false
```

Reply:
duh -_- :) thanks!